src/odemis/driver/symphotime.py

```diff
@@ -233,7 +233,7 @@ def to_bytes(self):
             for val in self.data:
                 output_string += struct.pack('I', val)
         elif self.typ == PQ_OPT_DATATYPE_FIXED_LENGTH_STRING:
-            output_string += struct.pack('<BH', self.typ, len(self.data)) + self.data
+            output_string += struct.pack('<BH', self.typ, len(self.data)) + self.data.encode('ascii')
```

Comment:
This may be used to encode non-ascii characters (eg, filename). I am not sure what's the expected encoding on the Symphotime server side, but I think utf-8 would be a good bet.
So, encode in "utf-8" and pass the length of the *encoded* data.